templates/readme_md-template-v2.md

```diff
@@ -0,0 +1,128 @@
+
+# Project Title
+
+One Paragraph of project description goes here. Include the copy: "<PROJECT NAME> is an [OASIS Open Project](https://oasis-open-projects.org/)"
+
+## Introduction
+
+* State what problem(s) your project solves.
+* Note its development status.
+* List the most useful/innovative/noteworthy features.
+* Note and briefly describe any key concepts (technical,  philosophical, or both) important to the user’s understanding.
+* Link to any supplementary blog posts or project main pages.
+* Compare/contrast your project with other, similar projects so the user knows how it is different from those projects.
+* Highlight the technical concepts that your project demonstrates or supports. Keep it very brief.
+* If possible, include screenshots and demo videos.
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See deployment for notes on how to deploy the project on a live system.
+
+### Prerequisites
+
+What things you need to install the software and how to install them. Also list any dependencies, if applicable.
+
+```
+Give examples
+```
+
+### Installing
+
+A step by step series of examples that tell you how to get a development env running
+
+Say what the step will be
+
+```
+Give the example
+```
+
+And repeat
+
+```
+until finished
+```
+
+End with an example of getting some data out of the system or using it for a little demo
+
+## Running the tests
+
+Explain how to run the automated tests for this system
+
+### Break down into end to end tests
+
+Explain what these tests test and why
+
+```
+Give an example
+```
+
+### And coding style tests
+
+Explain what these tests test and why
+
+```
+Give an example
+```
+
+## Deployment
+
+Add additional notes about how to deploy this on a live system
+
+## Built With
+
+* [Dropwizard](http://www.dropwizard.io/1.0.2/docs/) - The web framework used
+* [Maven](https://maven.apache.org/) - Dependency Management
+* [ROME](https://rometools.github.io/rome/) - Used to generate RSS Feeds
+
+## Contributing
+
+Please read [CODE_OF_CONDUCT](link to your code_of_conduct.md file) and [CONTRIBUTING.md](link to your contributing.md file) for details on our code of conduct, and the process for submitting pull requests to us.
+
+
+## Versioning
+
+We use [SemVer](http://semver.org/) for versioning. For the versions available, see the [tags on this repository](https://github.com/your/project/tags).
+
+## Authors &/or TSC & PGB members
+
+* **Billie Thompson** - *Initial work* - [PurpleBooth](https://github.com/PurpleBooth)
+
+See also the list of [contributors](https://github.com/your/project/contributors) who participated in this project.
+
+## Join us on Slack
+If your project has a Slack, add it here, eg. "Click here and fill out the form to receive an invite to the [Project name] slack instance, to meet the community and learn more about how you can engage."
+
+## License
+
```

Comment:
Add link to the approved licenses -> https://www.oasis-open.org/policies-guidelines/open-projects-process/#repository-specification-licenses